content/changes.md

```diff
@@ -8,6 +8,53 @@ summary: 'Learn about new features, fixes, and other improvements'
 
 Learn about notable new features, fixes, and other improvements to the application!
 
+## May 31, 2024 
+
+### Features 
+
+- **New Versioning Embed App**
+
+  Our new versioning embed app, released by NPM packages, provides our customers with a way to introduce improvements without affecting their current use cases ensuring consistency and quality. Visit our [embed repository](https://github.com/qualified/embed-demos) to learn more. 
```

Comment:
The main value proposition of the standalone Qualified Embed SDK package is that customers can use it in modern build systems, such as in React projects, by installing it with npm rather than a script tag.

It's nice too that they can pin to a version since this frees us up to make more interesting improvements down the line, but this is more of a source of value for us than it is for them at the moment.

This is a fairly significant and exciting announcement, so we can go into more depth here and really try to make a splash.

```suggestion
    We've released our standalone browser embed.js script as an [open source npm package](https://www.npmjs.com/package/@qualified/embed). This lets you easily integrate it into your application's modern build system, while facilitating package fixes, improvements and breaking changes.

   We'll continue to support the legacy standalone script and documentation indefinitely, but we encourage teams using Embed to upgrade to a non-legacy version to benefit from the build and be prepared for forthcoming improvements.

   Check out the [npm package](https://www.npmjs.com/package/@qualified/embed) and see it in action in our [React, Webpack and Embed demo](https://github.com/qualified/embed-demos/tree/master/demos/react).
```

Reply:
I prefer to change the term "indefinitely" for "for now" we can not compromise to continue supporting anything forever.

Reply:
That's fine too, but I don't think "indefinitely" implies forever, just "for the forseeable future". Eventually we'll reimagine and rewrite embed and probably stop supporting the whole package, but that's probably not for 4 or 5 (or more) years, and nobody will fault us for doing that.

"Indefinitely" evokes more trust than "for now", suggesting we won't pull the rug out from under anyone, but I'm flexible on this so use your judgment.